contribution.md

```diff
@@ -0,0 +1,113 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+[Code of Conduct](#code-of-conduct) 
+
+# Pre-Requirements
+To contribute to our community project,please download and install the following:
+
+1.  [Vagrant](https://www.vagrantup.com/)
+2.  [VirtualBox](https://www.virtualbox.org/)
+3.  [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+### Run the project on your local machine
+
+A step by step series that tell you how to get Pro4U up and running
+
+1. Clone the repository
+2. Open command prompt in Pro4U directory
+3. Spin the environment using `vagrant up` 
+4. That's it! your virtual machine is up and running 
+5. Go to http://localhost:8000 in your web browser
+6. You can enter the virtual machine by running `vagrant ssh`
+7. When finished, run `vagrant destroy -f` to tear down the environment\
+**_NOTE:_** By running this command all changes on the virtual machine will be deleted.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the CHANGELOG file with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!
+4.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists, **If not CREATE it!**
+5.	Make sure your code is conformed to the PEP8 standard
+
+**Note: for fast maintainers review please refer Reviewing PRs**
```

Comment:
Keep terminology consistent - stick to "Reviewers" and "Maintainers", no "fast reviewers")